TRANSLATION.md

```diff
@@ -0,0 +1,77 @@
+# reactjs.org saytining O'zbek tilidagi tarjimasi
+
+Hozirgi jarayon: 
+https://github.com/reactjs/uz.reactjs.org/issues/1
+
+## Tarjima bo'yicha kelishuv
+
+| Terminning asli | Tarjim |
+| ------------------ | ---------- |
+| API reference | API ma'lumotnoma |
+| array | massiv |
+| arrow function |  arrow funksiya, strelka funksiya  |
+| attribute | atribut |
+| batch | yangilash guruhi |
+| batching | guruhlash |
+| bug | bag, xatolik  |
+| bundler | bandler |
+| callback | kolbek |
+| camelCase | *camelCase* |
+| class component | sinf komponentasi, class komponent |
+| (un)controlled component | boshqaril(may)adigan komponent |
+| debugging | debag qilish |
+| DOM container | DOM-konteyner |
+| error | xatolik |
+| feature | imkoniyat |
+| development | dasturlash |
+| development mode | dasturlash rejimi |
+| framework | freymvork |
+| function component | funksiyali komponent |
+| hook | ho'k, ilgak |
```

Comment:
I think it's better to use only "uzbeknglish" version

Reply:
what do you mean? could you explain again

Reply:
keep only ho’k

Reply:
keep only ho’k

Reply:
I just want to emphasize that Hook is Capitan Ho'k's name. 

O'zbek tilida: Kapitan Ho'k ni nomlanishi ya'ni qo'lidagi ilgak bilan nomlanganligini ta'kidlamoqchi edim. Ho'k uzbek tiliga ILGAK deb tarjima qilinadi.

Reply:
@turdiyev oh, I guess you can add note about this in hooks section https://reactjs.org/docs/hooks-intro.html